MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include README.md LICENSE
+graft pybind11
```

Comment:
Hi, nice work, but I think you are including to much of pybind11. Since this is a cmake project, I think you only need the headers, the CMakeLists.txt and the *.cmake files that you get further below with the global-include.

Reply:
Adding to that, the LICENSE should probably be also included...

Reply:
Good point. Docs and tests are not needed! I would still do it by folder, though - /tools, /include, and a few files in the main directory.